### ISSUE TITLE
Example submission: TIME BASE ERROR

### ISSUE THREAD

**Issue by drodz11** [bavc/avaa#214]
Video of time base error captured from VHS. AVAA page: https://bavc.github.io/avaa/artifacts/time_base_error.html 

Link to video clip (1.2MB): https://drive.google.com/file/d/1q5RnRi5K4QwSEgDr5ayWoelj_2BUAiNA/view?usp=sharing

**Comment by ablwr** [comment]
Adding people who I believe have the keys to the host site/account for media assets.

**Comment by ablwr** [comment]
And thanks a bunch Dave!!

**Comment by iamdamosuzuki** [comment]
Hi! I just want to verify that this is meant to be an example of flagging, which is present at the very beginning of the clip. 

This is a little complicated because there is an entry for "Time Base Error" which mentions flagging here: https://bavc.github.io/avaa/artifacts/time_base_error.html

but there is also an entry for "Skew" which is "also known as: Flagging" here: https://bavc.github.io/avaa/artifacts/skew_error.html

The problem is that the example image in the Time Base Error page makes a distinction between Skew and Flagging, which is pretty subtle but I honestly think it's worth preserving the distinction. Here's my proposed solution:

- Add a new entry for "Flagging"
- Move the example from "Skew" to "Flagging"
- Add this new example to "Flagging"
- Add links from Time Base Error to "Skew" and "Flagging"

Whaddya think?

**Comment by drodz11** [comment]
Hi Morgan!

So I think the compression in Google Drive must have squashed out the error. In playing it through the viewer I provided in the link I see only a brief glimpse of the error at the start of the video, however playing the clip on my local machine using VLC I see it predominantly throughout the entire clip.

I tried uploading to Vimeo and got a better result: https://vimeo.com/351246350. Please take a look and let me know if you still think it fits this new categorization. If so, I say go ahead with your proposed changes. Let me know if you need anything else on my end and thanks!

**Comment by ablwr** [comment]
The Stanford folks I tagged on this have access to an archive.org account where we can host that media and hopefully retain the errors that we are trying to present here! I don't have access or credentials to that account, however.

**Comment by ablwr** [comment]
I like that solution, thanks @iamdamosuzuki !

**Comment by oceanofsound** [comment]
I'm actually not sure I have the log in details for the archive.org account, although it's possible the credentials never made it into my password manager. Soundcloud, on the other hand, I do have. @bturkus, was it you who originally did all the video uploading to the IA?

**Comment by bturkus** [comment]
not to pass the buck, but it was @kellyhaydon who added most fo the vids, and I think @iamdamosuzuki and BAVC are the keepers of the log in info (tho I bet I could guess it lol).

Thanks for adding something new Dave, and sorry for the logistics issues.

**Comment by drodz11** [comment]
@bturkus Hey no problem! Happy to be contributing to this project. 

@iamdamosuzuki  @kellyhaydon Just to ensure it didn't get lost in the back and forth: the better video that actually shows the error can be found here:  https://vimeo.com/351246350, not in the initial Google doc link I provided. Thanks and let me know if y'all need anything else on my end.

**Comment by bturkus** [comment]
To pick up on the @iamdamosuzuki discussion, personally I'm not sure if it is worth preserving the distinction between flagging (top of screen) and skew (bottom). I've never thought of skew as a bottom-only phenomenon, more I've thought that skew was the cause, and flagging the symptom. 

I don't have a Compendium of Image Errors handy, but in Practical VCR Repair I don't believe that David Ronan mentions flagging specifically. For skew, he defines it as: 

`Any undesirable slant or twist to tape along the tape path, such as tape angling upward or downward instead of passing straight between capstan and pinch roller. Also, error in tape tension, causing a change in position, size, or shape of video tracks on tape between record and playback.`

And in VCR troubleshooting and Repair, Brenner and Capelo also refrain from a specific glossary mention from flagging, defining skew as:

`The change in size or shape of video tracks from the time of recording to the time of playback. When present, this usually results from poor tension caused by misalignment or ambient conditions that affect the tape or tape path. The problem appears as as a TV image that is bent out of shape.`

And a skew error as:

`An error describing horizontal sync and video signal advance or retard from normal timing.`

From my perspective, I think it would make the most sense for time-base error to be the overarching category, with skew error (aka flagging, aka hooking distortion, which is how every Sony manual that I've picked up describes it) as a subcategory with a direct link. And within the skew error entry, it'd be awesome if we could elaborate on what the skew controls in a VTR are actually controlling (this I don't fully know, but am digging into it now).

Anyone else have any thoughts about these?

Ben

**Comment by iamdamosuzuki** [comment]
@drodz11 You're right, that vimeo version shows the error MUCH better. google drive seems to enjoy doing whatever it wants to video files. Is there a chance you could send me the original video file through some other sort of cloud delivery method? I can upload it to the internet archive from there. 

@bturkus Thanks so much for such an informative post. I think it would be possible to condense most of this info int a single entry. The architecture of AVAA is built for this sort of thing (tags, and "aka" are really great). However, I think there's a bit of discontinuity in that Ronan describes skew as a physical tape-path issue, where as the inclusion of the Global Village image in the TIME BASE ERROR entry implies that it's a time base error. 

Maybe it's time we revamp the TIME BASE ERROR page? We could caption the Global Village flyer to say that skew doesn't necessarily mean it's at the bottom of the image, and then build out the rest of the entry to discuss the subtle differences between the different types of errors?

**Comment by michaelangeletti** [comment]
Looking at the artifact example in Vimeo, it's possible that this is related to a [dihedral problem](https://bavc.github.io/avaa/artifacts/dihedral_maladjustment.html) because of the way the flagging is shaped and the way that it is moving. 

@bturkus I love your citations. The Compendium has synonyms for skew error as:  skewing, skew maladjustment, flagging, flag waving, and faulty tape tension regulator.

@iamdamosuzuki That sounds like a good approach. I see time base error as a broad general term that encompasses a lot of different types of artifacts. So I think it might make sense to continue to give certain very common artifacts potentially related to time base error (e.g., flagging, skew) their own AVAA entries to make them more readily accessible to users, and then link to the time base error entry.

Using time base error in a more generic sense, I mostly think of intraframe artifacts with horizontal timing misalignment. The sub-categories of flagging and skew are types of TBE that may be related to tape tension or other things. Personally, the generic time base error I usually refer to is describing that horizontal timing misalignment due to the mechanical differences between a record deck and a playback deck. It's harder to find good examples of this because in its severe natural state, digitization hardware doesn't like it (in part, probably because there is also an unstable vertical timing as well that causes digitization problems). If you put it through a TBC so it can be digitized well, it changes the way the artifact looks (usually improves/eliminates it) and then it becomes less useful as an example. It would be great to find a high resolution example in a good quality dub, where it is recorded into the image.

I think it gets convoluted because a time base error artifact may be the result of a record deck that was misaligned or otherwise faulty, stiction, tape shrinkage or expansion, pinch roller or guide problems, other tape path problems, shell problems, or it could be recorded into the image. So I think there needs to be a lot of clarity in the cause/effect relationship stated in the entries.

**Comment by drodz11** [comment]
@iamdamosuzuki There should be an option to download the video on the Vimeo page. If this isn't working for whatever reason, here's the video in Dropbox: https://www.dropbox.com/s/ajftz8nnti6ujaq/TBE_forAVAA.mp4?dl=0

@bturkus @michaelangeletti thanks for all this super interesting information! FWIW, the capture setup I am using is a Panasonic AG-1980 SVHS deck (TBC is built in) running into an Intensity Shuttle/vrecord. The series of tapes that this error came out of were produced in '89/'90 at Carolina Coastline Community College. Of the 13 tapes I digitized in this series, this was the only one to show the artifact. This leads one to think that it is a product of some distortion or issue with the tape itself as opposed to the equipment but I wouldn't call that a hard and fast conclusion.

Let me know if there's any further info I can provide.
